src/app/public/modules/fluid-grid/fluid-grid-gutter-size.ts

```diff
@@ -0,0 +1,5 @@
+export enum SkyFluidGridGutterSize {
+  Small = 5,
```

Comment:
Baking the margin values into the enum is immediately helpful (when generating the appropriate margin style), but it locks the value into the business logic instead of the stylesheet. For example, if we needed to adjust the `Small` margin to something like `4px`, we couldn't change the `enum` because it would be a breaking change. I would instead have the `enum` represent a generic "decision" for the desired margin, but then translate the enum into a specific CSS class so that the styles are completely untethered to the business logic.

```
.sky-fluid-grid-gutter-size-small {
  padding-left: 5px;
  padding-right: 5px; // This value can now be changed without updating the enum value!
}
```

Reply:
Damn. That's an excellent thought. I should have considered this. Great catch! I'll get to work on it.

Reply:
@Blackbaud-SteveBrush this is ready for you again. Great suggestion - this made the code much cleaner.